ckanext/scheming/plugins.py

```diff
@@ -1,6 +1,8 @@
 #!/usr/bin/env python
 # encoding: utf-8
 
+from future import standard_library
+standard_library.install_aliases()
```

Comment:
is this necessary? `install_aliases` does a ton of monkey-patching and module re-loading, would be nice to avoid if we can.

Reply:
it enables `urllib` alias for python2 - https://github.com/ckan/ckanext-scheming/pull/230/files/696c4295343925925bb18bfe580c04714a0c8fd4#diff-83fc01bdddb5a3e63073f84bc5f67fa1R388

I'll change it to `six` version  - looking on it from the perspective of all migrated extension, now i see, that it probably would be better not to use `future` library as almost everything available in six or via some polyfill.

Reply:
Thanks! When I was testing install_aliases in another package it seemed like it caused more problems than it solved.

Reply:
Unfortunately, i've just got another proof of your words. [This issue in ckanext-dcat](https://github.com/ckan/ckanext-dcat/issues/179) was caused by using `future.install_aliases` :sob: 
Dev from our team was going to create PR with fix into ckanext-dcat, and I have to update all my pending PRs into extensions, removing `future`.